aptos-move/aptos-release-builder/data/release.yaml

```diff
@@ -116,6 +116,7 @@ proposals:
       discussion_url: "https://github.com/aptos-foundation/AIPs/issues/331"
     execution_mode: MultiStep
     update_sequence:
+      - RawScript: data/proposals/aip_67_initialization.move
```

Comment:
I think you need aptos-move/aptos-release-builder/data/proposals/aip_67_initialization.move full path cause release builder would be running on aptps-core directory.

Reply:
fixed